README.md

```diff
@@ -170,20 +170,28 @@ Flags:
 
 ## Roadmap
 
-* Additional language support for just-in-time (JIT) compilers, and dynamic languages (non-exhaustive list):
-  * Ruby
+* Additional **zero-instrumentation** language support for just-in-time (JIT) compilers and dynamic languages (non-exhaustive list):
+  * [JVM](https://www.parca.dev/docs/java-support)
+  * Lua
   * Node.js
-  * Python
-  * JVM
+
+
+  > [!IMPORTANT]
+  > All JIT compiled languages with frame pointer, and [Perf map](https://github.com/torvalds/linux/blob/master/tools/perf/Documentation/jit-interface.txt) or [jitdump](https://github.com/torvalds/linux/blob/master/tools/perf/Documentation/jitdump-specification.txt)
```

Comment:
This sentence feels incomplete, but might be me!

Reply:
Also, I feel this is too "technical" if this makes sense, we should point users to more user-friendly documentation. Most people enabling the flags for JIT symbolization probably don't know about the format